actions/workflows/gather_ngi_pipeline_reports_workflow.yaml

```diff
@@ -22,14 +22,16 @@ workflows:
                 summary_user: <% task(get_config).result.result.summary_user %>
                 summary_ngi_pipeline_reports_destination: <% task(get_config).result.result.summary_ngi_pipeline_reports_destination %>
                 irma_reports_remote_path: <% task(get_config).result.result.irma_reports_remote_path %>
+                irma_user: <% task(get_config).result.result.irma_summary_ssh_user %>
+                irma_key: <% task(get_config).result.result.irma_summary_ssh_key %>
                 summary_host_key: <% task(get_config).result.result.summary_host_key %>
               on-success:
                 - rsync_from_irma
 
             rsync_from_irma:
               action: core.remote
               input:
-                cmd: rsync -e "ssh -i /home/seqsum/.ssh/mm-xlas002" -r funk_901@irma1.uppmax.uu.se:<% $.irma_reports_remote_path %>/ <% $.summary_ngi_pipeline_reports_destination %>/
+                cmd: rsync -e "ssh -i <% $.irma_key %>" -r <% $.irma_user %>@irma1.uppmax.uu.se:<% $.irma_reports_remote_path %>/ <% $.summary_ngi_pipeline_reports_destination %>/
```

Comment:
perhaps make the irma host configurable as well while we're at it?

Reply:
done :)